apps/submit-app.md

```diff
@@ -6,26 +6,121 @@ title: Submit an app
+
+Tick the respective check box(es) to set up the scope for your app.
+This indicates the type of calls your app will make on the REST API.
+
+#### App details
+
+English app details are mandatory.
+If a merchant switches his shop to a language your app does not support, the fallback language is English.
+If you decide to offer your app in multiple languages, tick the respective check boxes.
+You will then also have to fill out all mandatory fields for the additional language(s).
+
+Requirements: describe the requirements for using your service, e.g. pricing.
+
+{% highlight text %}
+Example: Subscription model: $19.99 per month.
+{% endhighlight %}
+
+Developer or company name: provide the name of the individual(s) or company which developed the app.
+
+Features: provide three key features of your app, which will be shown as an list.
+Each item is limited to 140 characters.
+
+{% highlight text %}
+Example:
+
+- Export your orders and print shipping labels.
+- Automatic synchronisation between your shop and the app.
+- Detailed statistics.
+{% endhighlight %}
+
+Short description: write a short description of your app with max. 160 characters.
+
+{% highlight text %}
+Example:
+
+Handle your shipping process with ease:
+Automatic order synchronisation and ready-to-print shipping labels.
+{% endhighlight %}
+
+Long description: write a detailed description of your app with max. 3000 characters.
+You can use [Markdown][markdown] for this.
+
+{% highlight text %}
+Example:
+
+Simplify your shipping process.
+No need for setting up shipping details manually any longer.
+With this app, all orders are automatically synchronised with your system
+and all shipping labels are ready-for-printing with ease.
+{% endhighlight %}
+
+Search terms: provide keywords that describe your app.
+Use a comma to separate the keywords.
+
+Application callback URL: make sure that this URL is SSL and public.
+After the merchant has confirmed the installation of your app in his store, it redirects the merchant to your application.
+The required query parameters will be added automatically.
+
+Open app URL: the target link of "Open app" on the app detail page after a successful installation.
+Provide a link to your most important interface or dashboard.
+
+Video ID: provide the ID for a video hosted on one of our supported providers.
+
+{% highlight text %}
+Example:
+
+YouTube
+https://www.youtube.com/watch?v={Id}
+{% endhighlight %}
+
+We will need logos and at least one screenshot to display your app in the App Store.
+
+App detail page: use 64x64px, GIF, JPEG, PNG.
+
+App store: use 260x160px, GIF, JPEG, PNG.
+
+Screenshots: use 800x600px - 1920x1080px, GIF, JPEG, PNG.
+
+Before submitting your app, please make sure you have read and understood our app submission guidelines.
+Review all the information that you provide to us and also make sure that the **Application Callback URL** and **Open app URL** are correct.
 
 # You have reached the home stretch!
```

Comment:
maybe change with "You are on the home stretch"
speak with Jess with what she thinks